python/unityagents/rpc_communicator.py

```diff
@@ -15,7 +15,8 @@
 
 
 class UnityToExternalServicerImplementation(UnityToExternalServicer):
-    parent_conn, child_conn = Pipe()
+    def __init__(self):
+        self.parent_conn, self.child_conn = Pipe()
```

Comment:
Is this the fix for the bug preventing using environments with python multi-processing.

Reply:
Yep. In the old code they were global variables, which prevented multiple instances of them from being created correctly.